app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -19,7 +19,7 @@
       <div class="row">
         <div class="col-9">
           <h5 class="fw-bold">
-          <% if feedback_message.feedback_type == "connect" %>
+          <% if feedback_message.offender %>
```

Comment:
Considering the previous code only checked for Connect, will `offender` ever be set in other cases? Or is this a functionality change?

Reply:
It is a functionality change (sort of), if we ever add a new kind of feedback message (or a new way to leave an abuse report) that captures a reporter and an offender, then that would be displayed, the same way connect feedback had been (still is). The only case where offender id was ever set was connect (there were three feedback types, connect, bug report, abuse report, both bug report and abuse report captured a url and a message, while connect feedback captured 2 involved parties and the message. I'm leaving the behavior in place, since we're not deleting the connect feedback here. 

I was treating the intent to preserve existing reports and display them correctly/completely as a constraint, but we definitely can relax that. We _could_ introduce a DUS to remove "connect" feedback, and remove this condition (since those are the only kinds of feedback messages that had this), and it's less code to maintain (knowing that there would be no such reports to show, there's no need to display them differently, and a lot of if/else blocks in the view collapse to a simple "display the data  for the object" flow.

Reply:
@djuber This seems like it may be worth a follow-up PR. Removing extraneous branches and DB column seems worth the effort to me.